Mathlib/Analysis/Normed/Ring/IsPowMulFaithful.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2024 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández
+-/
+import Mathlib.Analysis.Normed.Algebra.Norm
+import Mathlib.Analysis.SpecialFunctions.Pow.Continuity
+
+/-!
+# Equivalent power-multiplicative norms
+
+In this file, we prove [BGR, Proposition 3.1.5/1][bosch-guntzer-remmert]: if `R` is a normed
+commutative ring and `f₁` and `f₂` are two power-multiplicative `R`-algebra norms on `S`, then if
+`f₁` and `f₂` are equivalent on every subring `R[y]` for `y : S`, it follows that `f₁ = f₂`.
+
+## Main Definitions
+* `AlgebraNorm.restriction`: The restriction of an algebra norm to a subalgebra.
```

Comment:
It seems this moved the the `Basic` file.

Reply:
Thanks! I have deleted this comment and the next.